content/docs/csidriver/features/powerstore.md

```diff
@@ -725,3 +728,5 @@ The following is the list of all the attribtues supported by PowerStore CSI driv
 **Note:**
 >Refer to the PowerStore array specification for the allowed values for each attribute, at `https://<array-ip>/swaggerui/`. 
 >Make sure that the attributes specified are supported by the version of PowerStore array used. 
+
+>This feature is supported only on helm installation.
```

Comment:
I think there is no need to repeat this note in this file, should be sufficient if we add it in vg-snapshotter file, Otherwise, we will have to again add this note in powerflex doc. as well, Let's keep it in one place.

Reply:
This section is for configurable volume attributes right

Reply:
ah ok, missed seeing the heading.